live_tests/modules/test_hspsmswrapper.py

```diff
@@ -0,0 +1,32 @@
+import time
+from cshsms.settings import HSPSMS_API, HSPSMS_USERNAME, HSPSMS_SENDERNAME, TEXTLOCAL_PHONENUMBER
+from django.test import TestCase
+from modules.hspsmswrapper import Hspsms
+from modules.texter import Texter
+
+HSPSMS = Hspsms(apikey=HSPSMS_API, username=HSPSMS_USERNAME, sendername=HSPSMS_SENDERNAME)
+class HspsmsSendingTests(TestCase):
+
+	def test_send_message_to_fake_number(self):
+		time.sleep(60)
+		fr = HSPSMS.send_transactional_message(message='Example Hspsms message to fake number', phone_number='0000000')
+		self.assertTrue(fr)
+		self.assertIsInstance(fr, list)
+		self.assertTrue(fr[0]['responseCode'])
+		self.assertEqual(fr[0]['responseCode'], "Message SuccessFully Submitted")
```

Comment:
Yes, "SuccessFully" is the real response we are supposed to get. That's not a typo.